roles/softserve/tasks/main.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: Create a user
+  user:
+    name: "{{ user }}"
+    state: present
+
+- name: Install required packages
+  package:
+    state: present
+    name: "{{ item }}"
+  with_items:
+  - python-virtualenv
+  - gcc
+  - git
+  - python-devel
+  - postgresql-devel
+  - mock
+
+- name: Create the project directory
+  file:
+    path: "{{ destination }}"
+    state: directory
+    owner: softserve
+    group: softserve
+    recurse: yes
+
+- name: Clone the code
+  git:
+    repo: 'https://github.com/gluster/softserve.git'
+    dest:  '{{ destination }}/code'
+    version: HEAD
+    update: yes
+    force: yes
+
+- name: Add tmpfiles snippet
+  template:
+    dest: /etc/tmpfiles.d/softserve.conf
+    src: softserve.tmpfile.conf
+  notify: Create systemd tmpfile
+
+- name: Add gunicorn systemd service
+  template:
+    dest: /etc/systemd/system/softserve.service
+    src: softserve.service
+  notify: Reload systemd
+
+- name: Creating and activating venv
+  pip:
+    chdir: "{{ destination}}/code"
+    virtualenv: "{{ destination }}/env"
+    virtualenv_site_packages: yes
+    name: setuptools
+    extra_args: --upgrade
+
+- name: Installing the dependencies
+  pip:
+    chdir: "{{ destination }}/code"
+    virtualenv: "{{ destination}}/env"
+    requirements:  "{{ destination}}/code/requirements-prod.txt"
+
+- name: Copy the config file to the softserve folder
+  template:
+    dest: "{{ destination }}/code/instance/application.cfg"
+    src: application.cfg
+
+- name: Database migration
+  command: FLASK_APP=manage.py flask db upgrade
+  args:
+    chdir: "{{ destination }}/code"
+
+- name: Configure Crontab
+  cron:
+    name: "shutdown check"
+    minute: "*/5"
+    job: "/srv/softserve/env/bin/flask shutdown_check"
+
+- name: Start the softserve service
+  service:
+    name: softserve
+    enabled: True
+    state: started
+
+- meta: flush_handlers
```

Comment:
We no longer need it if that's the last thing in the playbook, sorry, seems my suggestion did result into 2 fix for 1 issue :/